src/sail_lean_backend/sail_plugin_lean.ml

```diff
@@ -154,26 +154,26 @@ let create_lake_project (out_name : string) =
   let project_dir = base_dir ^ "/" ^ out_name in
   Unix.mkdir project_dir 0o775;
   (* Create .gitignore file *)
-  let gitignore = open_out (project_dir ^ "/.gitignore") in
+  let gitignore = open_out (Filename.concat project_dir "/.gitignore") in
   (* Ignore the `z3_problems` file generated by Sail and the `.lake` directory generated by lake*)
   output_string gitignore "/.lake\nz3_problems";
   close_out gitignore;
   (* Create the `lean-toolchain` file *)
-  let lean_toolchain = open_out (project_dir ^ "/lean-toolchain") in
+  let lean_toolchain = open_out (Filename.concat project_dir "/lean-toolchain") in
   (* Set the Lean version *)
   output_string lean_toolchain ("leanprover/" ^ lean_version);
   close_out lean_toolchain;
   (* Camel-case the output name *)
```

Comment:
I think some of the comments might not be needed as the code is self-explanatory enough